### PR TITLE
Use pigz to compress instead of gzip if available

### PR DESCRIPTION
**Pull request by sxa555**
Speeds up the native arm32 builds significantly if pigz is available. Won't make such a difference to other platforms but shouldn't cause any negative effects, and will fall back to gzip if pigz is unavailable

### CODE REVIEW

File: sbin/build.sh [hunk 1]
```diff
@@ -459,8 +461,8 @@ createOpenJDKTarArchive()
   echo "OpenJDK repo tag is ${OPENJDK_REPO_TAG}. JRE path will be ${JRE_TARGET_PATH}"
 
   if [ "$USE_DOCKER" == "true" ] ; then
-    GZIP=-9 tar -czf OpenJDK.tar.gz ./"${OPENJDK_REPO_TAG}"
-    GZIP=-9 tar -czf OpenJRE.tar.gz ./"${JRE_TARGET_PATH}"
+    GZIP=-9 tar --use-compress-program=$COMPRESS -cf OpenJDK.tar.gz  ./"${OPENJDK_REPO_TAG}"
```
Comment (karianna) [inline sbin/build.sh]: unnecessary whitespace?

File: sbin/build.sh [hunk 2]
```diff
@@ -473,12 +475,12 @@ createOpenJDKTarArchive()
         zip -r -q OpenJRE.zip ./"${JRE_TARGET_PATH}"
       EXT=".zip" ;;
       aix)
-        GZIP=-9 tar -cf - ./"${OPENJDK_REPO_TAG}"/ | gzip -c > OpenJDK.tar.gz
-        GZIP=-9 tar -cf - ./"${JRE_TARGET_PATH}"/ | gzip -c > OpenJRE.tar.gz
+        GZIP=-9 tar -cf - ./"${OPENJDK_REPO_TAG}"/ | $COMPRESS -c > OpenJDK.tar.gz
+        GZIP=-9 tar -cf - ./"${JRE_TARGET_PATH}"/  | $COMPRESS -c > OpenJRE.tar.gz
```
Comment (karianna) [inline sbin/build.sh]: whitespace again

File: sbin/build.sh [hunk 3]
```diff
@@ -473,12 +475,12 @@ createOpenJDKTarArchive()
...
       EXT=".tar.gz" ;;
       *)
-        GZIP=-9 tar -czf OpenJDK.tar.gz ./"${OPENJDK_REPO_TAG}"
-        GZIP=-9 tar -czf OpenJRE.tar.gz ./"${JRE_TARGET_PATH}"
+        GZIP=-9 tar --use-compress-program=$COMPRESS -cf OpenJDK.tar.gz  ./"${OPENJDK_REPO_TAG}"
```
Comment (karianna) [inline sbin/build.sh]: whitespace again

File: sbin/build.sh (hunk 2 above)
Comment (sxa555) [reply sbin/build.sh]: In this case I just lined it up with the line above to make it look better but happy to change it back
